classes/01class/exercises/c01-aws01/roperto/README.md

```diff
@@ -0,0 +1,36 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```shell script
+aws ec2 create-key-pair --key-name devops-test-key --query 'KeyMaterial' --output text > ~/.ssh/devops-test-key.pem
+aws ec2 create-security-group --description "Allow SSH from anywhere" --group-name "test-allow-ssh"
+aws ec2 authorize-security-group-ingress --group-id sg-074a6bea5b3ad2e6d --protocol tcp --port 22 --cidr 0.0.0.0/0
```

Comment:
Ideally, you would have a single `/32` IP here as source so you can reduce the exposure of your public instance. I understand that your home IP might be dynamic and you might have preferred just using 0.0.0.0

Reply:
Good suggestions, done. It will not allow anyone in the private network to reach that IP but we can consider that not a requirement.